gtk4-macros/src/util.rs

```diff
@@ -0,0 +1,82 @@
+use anyhow::{bail, Result};
```

Comment:
I would avoid adding a dependency on anyhow here and use a proper error type

Reply:
glib-macros also depends on anyhow so :man_shrugging:

Reply:
Ah right, well, it doesn't matter then